### ISSUE TITLE
Canceling Out Of Import Codes Dialog Results In Blank Unresponsive App

### ISSUE THREAD

**Issue by ikeaRobes** [ente-io/ente#2126]
### Description

TLDR; attempting to cancel out of import flow causes app to be blank / unresponsive until restarting it

iPhone 13 Mini iOS 17.5.1

_**steps to reproduce:**_
- launch the app
- hamburger menu
- Data > Import codes
- tap any import option ( Bitwarden, 2FAS, etc )
- tap cancel on the popup dialog
- observe a checkmark appears beside the import option as if it attempted to import
- press the X top right to exit import codes flow
- observe a blank screen
- restart app to restore functionality 

export of logs below

[ente-logs-2024-Jun-13-15-9.zip](https://github.com/user-attachments/files/15822762/ente-logs-2024-Jun-13-15-9.zip)

### Version

3.0.6

### What product are you using?

Ente Auth

### What platform are you using?

Mobile - iOS

**Comment by prateekmedia** [comment]
Will be fixed in latest version, X button is removed as it brings inconsistency with the UI.
